pkg/instancegroups/instancegroups.go

```diff
@@ -65,6 +67,32 @@ func NewRollingUpdateInstanceGroup(cloud fi.Cloud, cloudGroup *cloudinstances.Cl
 	}, nil
 }
 
+// User input routine copied from vendor/google.golang.org/api/examples/gmail.go
+func PromptInteractive(upgradedHost string) (stop_prompting bool) {
+	stop_prompting = false
+	reader := bufio.NewReader(os.Stdin)
+	glog.Infof("Pausing after finished %q", upgradedHost)
+	fmt.Printf("Continue? (Y)es, (N)o, (A)lwaysYes: [Y] ")
+	val := ""
+	var err error
+	if val, err = reader.ReadString('\n'); err != nil {
```

Comment:
Can we use the endline option in golang that is platform independent?  Eventually windows yay.

Reply:
Switched from using bufio Reader to bufio Scanner.